src/packet_analysis/Manager.cc

```diff
@@ -171,3 +182,67 @@ void Manager::DumpPacket(const Packet *pkt, int len)
 
 	run_state::detail::pkt_dumper->Dump(pkt);
 	}
+
+class UnknownProtocolTimerTimer final : public zeek::detail::Timer {
```

Comment:
Shouldn't that be `UnknownProtocolTimer`?

Reply:
Whoops.